usaspending_api/awards/v2/data_layer/orm.py

```diff
@@ -564,6 +571,55 @@ def fetch_psc_hierarchy(psc_code: str) -> dict:
     return results
 
 
+def fetch_disaster_emergency_fund_codes_for_award(award_id):
+    return list(
+        FinancialAccountsByAwards.objects.filter(award_id=award_id, disaster_emergency_fund__isnull=False)
+        .order_by("disaster_emergency_fund__code")
```

Comment:
is this intermediate `order_by` necessary?

Reply:
No.  Thanks.  Fixed.